indexer/src/main/java/org/pdxfinder/dataloaders/LoadHCI.java

```diff
@@ -308,7 +308,7 @@ void step17LoadModelDosingStudies() throws Exception {
     }
 
     @Override
-    void step18SetAccessGroup() {}
+    void step18SetAdditionalGroups() {}
```

Comment:
If a method needs to be overridden but is not implemented in the concrete class, you should use inside it : 
```throw new UnsupportedOperationException();```
(See [LoadIRCC step01GetMetaDataFolder override](https://github.com/PDXFinder/pdxfinder/blob/621fb24a950a6e87594b86f62859de83a16bb75d/indexer/src/main/java/org/pdxfinder/dataloaders/LoadIRCC.java#L107)), do the same for other classes where you've overridden it.